src/supertux/shrinkfade.hpp

```diff
@@ -24,18 +24,23 @@
 class ShrinkFade final : public ScreenFade
 {
 public:
-  ShrinkFade(const Vector& point, float fade_time);
+  enum Direction { FADEOUT, FADEIN };
+
+public:
+  ShrinkFade(const Vector& point, float fade_time, int draw_layer, Direction = FADEOUT);
```

Comment:
I suggest having the previous value, used in `draw()`, which is `LAYER_GUI + 1`, as a default.

Reply:
Done.